graylog2-web-interface/src/logic/ObjectHelper.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+// eslint-disable-next-line import/prefer-default-export
+export const trimObjectFields = (obj: object, fieldsToTrim: string[]) => {
```

Comment:
I would move this function inside the `ObjectUtils` file we have inside `util` folder.

Reply:
Thx I was searching for smth like that